test/unit/ready.js

```diff
@@ -1,18 +1,18 @@
-module( "event" );
+QUnit.module( "ready" );
```

Comment:
How does the dropdown from the screenshot at #2437 look like now? Is the double `event` entry changed to `event` & `ready`? Does choosing `ready` work?